### PR TITLE
COL-650: Fixes the links to the 'Google Form hosted' LiveSync feedback form 

### PR DESCRIPTION
**Pull request by franrob-projects**
This PR:

- Fixes the links to the 'Google Form hosted' LiveSync feedback form.

[COL-650: Fix feedback form link](https://ably.atlassian.net/browse/COL-650)

### CODE REVIEW

**Review by Srushtika — APPROVED** [review]

File: content/livesync/connector/index.textile [hunk 1]
```diff
@@ -17,7 +17,7 @@ docker pull ghcr.io/ably-labs/adbc:latest
 ```
 
 <aside data-type='note'>
-<p>Please "reach out":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewformif you have any feedback or prefer to use an Ably-hosted version of LiveSync.</p>
+<p>Please "reach out":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform you have any feedback or prefer to use an Ably-hosted version of LiveSync.</p>
```
Comment (m-hulbert) [inline content/livesync/connector/index.textile:20]: I think the "if" should have been part of the sentence (in both places) 🙂

**Review by m-hulbert — CHANGES_REQUESTED** [review]

File: content/livesync/connector/index.textile (hunk 1 above)
Comment (franrob-projects) [reply content/livesync/connector/index.textile:20]: Ah

Comment (franrob-projects) [reply content/livesync/connector/index.textile]: @m-hulbert I've addressed your feedback here: ecf2611848db5a62f6740d9d2173eb6357b6a274

**Review by m-hulbert — APPROVED** [review]
Good to go after a rebase 👍

**Pull request merged**